packages/ui/src/mocks/server.ts

```diff
@@ -52,8 +52,8 @@ export const fixAssociations = (server: Server<AnyRegistry>) => {
 
   const forumCategoryModel = schema.modelFor('forumCategory')
   // "Mirage: The forum-category model has multiple possible inverse associations for the forum-category.parent association."
-  forumCategoryModel.class.prototype.associations.parent.opts.inverse = 'forumcategoryparent'
-  forumCategoryModel.class.prototype.associations.forumcategoryparent.opts.inverse = 'parent'
+  forumCategoryModel.class.prototype.associations.parent.opts.inverse = null
+  forumCategoryModel.class.prototype.associations.forumcategoryparent.opts.inverse = null
```

Comment:
@jodator, @p-sad in my understanding `forumCategoryModel` is all the parents/ancestors of a category and not the list not the subcategories is this correct ?

Reply:
AFAICS they are stored in the same "table" so the relations are both ways.

So the field `parent` will be the direct parent. While `forumcategoryparent` is the other direction of the same relation. So it will hold "subcategories". Naming is automatic probably so it looks "weird".

So Having categories:

*   TopCategory(id=1)
    *   Sub A(id=2)
        *   SubASub1 (id=4)
        *   SubASub2 (id=5)
    *   Sub B(id=3)

The fields would be:

```
TopCategory
  parent: null
  forumcategoryparent:[2, 3]

Sub A
  parent: 1
  forumcategoryparent:[4,5]

Sub A Sub 1
  parent: 2
  forumcategoryparent:[]
```

Reply:
@thesan & @p-sad  I don't see if this change breaks something (fetching related categories) so it should be safe.

However, if you see that some of the subcategories/parents is not fetched, please see into that.

Reply:
I see. So it shouldn't be used to get the root categories then :disappointed:. Now it's only working because I implemented the `forumcategoryparent` as a list of parents.

Reply:
OH.. but using `parent` there should work. I was sure I was testing that with `parent` but, no :D

Reply:
OH.. but using `parent` there should work. I was sure I was testing that with `parent` but, no :D

Reply:
A simple `null` will do the trick:

```graphql
query {
  forumCategories(where:{parent:null}) {
    id
    title
    parentId
  }
}
```

Reply:
Can I ask you to fix that later and add @p-sad as a reviewer?

Reply:
Got it, I will do that. Thanks